marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/previewMatchingActivity.api

```diff
@@ -14,6 +14,10 @@
   {
     "name": "stepName",
     "datatype": "string"
+  },
+  {
```

Comment:
I think this is more intuitive, but I'd check with some other engineers to see if a description is needed here to fully clarify what the impact of true/false is for this parameter. They may have a better suggestion than "restrictToUris" as well.